assets/src/less/admin.less

```diff
@@ -300,6 +300,13 @@
     background: #fff;
 }
 
+.dokan-settings-wrap .metabox-holder form .form-table th {
+  width: 235px;
```

Comment:
Try to use ( em or rem ) unit instead of px. This would be better for responsiveness.